masterlist.yaml

```diff
@@ -2921,25 +2923,7 @@ plugins:
       #   - 00037EE9
       - crc: 0xB63A3D6B
         util: SSEEdit v3.2.1
-  - name: 'TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp'
```

Comment:
It's fine to keep the plug-ins in separate. 
The above inc: will only work for `TheWulfPanda - City Teleport Spells.esp`
and the dirty plug-in info will not show for `TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp`

Reply:
Ah okay, I thought I could squash it together into a single list entry. 😄 
I'm going to rebase my PR.